src/main/python/ttconv/isd.py

```diff
@@ -36,6 +36,23 @@
 import ttconv.model as model
 import ttconv.style_properties as styles
 
+class SignificantTimes:
+  def __init__(self, sig_times, cache = None):
```

Comment:
Could you add type hints here, please? 🙏

Reply:
Fixed